frontend/packages/kubevirt-plugin/src/components/boot-order/add-item-form-select.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { FormSelect, FormSelectOption } from '@patternfly/react-core';
+import { deviceKey, deviceLabel } from './constants';
+
+export const AddItemFormSelect: React.FC<AddItemFormSelectType> = ({
+  options,
+  label,
+  addByKey,
+}) => (
+  <FormSelect
+    value=""
+    id="boot-order-select"
```

Comment:
maybe we should pass these as a parameter or a default parameter?

Reply:
now a required parameter :+1: